.github/workflows/release.yaml

```diff
@@ -157,3 +157,19 @@ jobs:
           allowUpdates: true
           prerelease: ${{ github.event.inputs.pre_release }}
           body: ${{ steps.resolve-release-notes.outputs.body }}
+          
+  update-tutorials:
```

Comment:
you need to wait for `create-releases` to finish.
add 
`needs: create-releases`